specification/semantic_conventions.md

```diff
@@ -27,3 +28,26 @@ All Splunk distributions of OpenTelemetry,
 | Attribute  | Type | Description  | Examples  | Required |
 |---|---|---|---|---|
 | `splunk.distro.version` | string | The version number of the Splunk distribution being used. | `1.5.0` | Yes |
+
+Note: this section does not apply to Real User Monitoring libraries, as they do not use the OpenTelemetry Resource.
+
+## Real User Monitoring Spans and Attributes
+
+The following attributes MUST be added to all spans emitted by Real User Monitoring (RUM)
+instrumentation libraries:
+
+| Name                  | Type   | Description                                                           |
+| ----                  | ----   | -----------                                                           |
+| `app`                 | string | The value of the `applicationName` property configured in `SplunkRum` |
+| `splunk.rumSessionId` | string | The RUM session ID                                                    |
+| `splunk.rumVersion`   | string | Version of the RUM library                                            |
```

Comment:
Shouldn't we deduplicate attribute keys if possible? So use `splunk.distro.version` here instead?

Reply:
Since this isn't an distribution of OpenTelemetry, I'm not sure we should use the `distro` tag in here. This is a pretty targeted solution for RUM, based on otel, and the spans we generate won't be showing up in the normal APM UI, so I don't know that consistency is all that crucial.

That being said, I totally just made up this attribute name, and would be totally fine changing it to something else. Having some attribute to track the version of the library seems very important, though, for customer support.

Reply:
How about `splunk.rum.version` and `splunk.rum.session_id`? I think that would be more in-line with the otel attribute naming strategy

Reply:
I'm all for that. Right now, the RUM instrumentation (which started in the browser) kind of has its own semantic conventions, not all that closely related to otel. IMO, we should converge on something at least otel-like as much as we can.

Reply:
> IMO, we should converge on something at least otel-like as much as we can.

Definitely! 💯 
I was hoping that this PR might be a start of this

Reply:
For the moment, though, the RUM backend & frontend are dependent on "splunk.rumSessionId" being the name of the attribute, so we can't change that.

Reply:
🤷